Explorer/Assets/DCL/SDKComponents/CameraTransform/Systems/SDKCameraTransformSystem.cs

```diff
@@ -0,0 +1,81 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using CRDT;
+using CrdtEcsBridge.Components;
+using DCL.CharacterCamera;
+using DCL.Diagnostics;
+using DCL.Optimization.Pools;
+using DCL.SDKComponents.CameraTransform.Components;
+using DCL.Utilities;
+using ECS.Abstract;
+using ECS.Groups;
+using ECS.LifeCycle;
+using ECS.Unity.Transforms;
+using ECS.Unity.Transforms.Components;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.SDKComponents.CameraTransform.Systems
+{
+    /// <summary>
+    ///     This system creates a new entity using the reserved SpecialEntitiesID.CAMERA_ENTITY in order to create a TransformComponent to be on sync with the camera
+    ///     This allows using engine.CameraEntity as a parent target in the SDK
+    /// </summary>
+    [UpdateInGroup(typeof(SyncedPostRenderingSystemGroup))]
+    [LogCategory(ReportCategory.CAMERA_TRANSFORM)]
+    public partial class SDKCameraTransformSystem : BaseUnityLoopSystem, IFinalizeWorldSystem
+    {
+        private readonly Dictionary<CRDTEntity, Entity> entitiesMap;
+        private readonly ObjectProxy<Entity> cameraEntityProxy;
+        private readonly World globalWorld;
+        private readonly IComponentPool<Transform> transformPool;
+        private readonly CRDTEntity sdkCameraEntity;
+        private Entity cameraEntityMirror;
+
+        private SDKCameraTransformSystem(World world, Dictionary<CRDTEntity, Entity> entitiesMap, ObjectProxy<Entity> cameraEntityProxy, ObjectProxy<World> globalWorldProxy, IComponentPool<Transform> transformPool) : base(world)
```

Comment:
There is no need to pass `cameraEntityProxy`: you can use static `SingleInstanceEntity CacheCamera(this World world)` in `Initialize`